src/Components/SignIn/SignIn.css

```diff
@@ -0,0 +1,89 @@
+.signIn-form {
+  display: grid;
+  grid-template-areas:
+    "emaillabel passlabel"
+    "email pass"
+    "btn1 btn2"
+    "test test";
+
+  grid-template-columns: 1fr 1fr;
+  column-gap: 1rem;
+  row-gap: 9px;
+}
+#test-login {
+  grid-area: test;
+}
+.form-label {
+  color: var(--white);
+}
+
+#email-label {
+  grid-area: emaillabel;
+}
+#email-input {
+  grid-area: email;
+}
+
+#pass-label {
+  grid-area: passlabel;
+}
+#pass-input {
+  grid-area: pass;
+}
+
+#form-btn1 {
+  grid-area: btn1;
+}
+#form-btn2 {
+  grid-area: btn2;
+  text-align: center;
+  font-size: 13.33px;
+}
+
+.form-container {
+  display: flex;
+  place-content: center;
```

Comment:
Suggestion: Either use class selectors or ID selectors everytime to keep the code consistent.